src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversions.cs

```diff
@@ -257,7 +257,7 @@ private static Conversion ToConversion(OverloadResolutionResult<MethodSymbol> re
             }
 
             //cannot capture span-like types.
-            if (!method.IsStatic && methodGroup.Receiver.Type.IsByRefLikeType)
+            if (!method.IsStatic && methodGroup.Receiver?.Type?.IsByRefLikeType == true)
```

Comment:
What changed from 15.4 that caused this regression? Was the type previously non-null or did we add this line which incorrectly assumed it was non-null?

Reply:
The entire `if` statement is new.